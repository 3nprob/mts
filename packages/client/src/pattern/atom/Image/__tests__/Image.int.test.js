import { regular } from './Image.int.story'

describe('<Image />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
