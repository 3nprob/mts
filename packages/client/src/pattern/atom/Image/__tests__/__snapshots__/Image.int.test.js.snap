// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> Render must match regular() 1`] = `
<div
  className=""
>
  <div />
  <picture>
    <source />
    <img />
  </picture>
</div>
`;
